### PR TITLE
Added LESS to compile CSS

### PR DESCRIPTION
**Pull request by jcbrmn**
Was able to add in less and a clean css plugin that minifies all of the css. Color variables are at the top of each less file. I just left them for now, but you can easily split those into a different file and in your styles.less file just make sure you call the variables file before the other less files.

Let me know if you have any questions.

### CODE REVIEW

**Comment by yann300** [comment]
Thanks for this PR!

File: npm-debug.log [hunk 1]
```diff
@@ -0,0 +1,49 @@
+0 info it worked if it ends with ok
```
Comment (yann300) [inline npm-debug.log]: please could you remove this file?

**Comment by jcbrmn** [comment]
No problem @yann300 That file has been removed.

**Comment by yann300** [comment]
please rebase

File: package.json [hunk 2]
```diff
@@ -8,7 +8,7 @@
     "browser-test-remote-firefox": "nightwatch --config nightwatch.js --env default",
     "browser-test-remote-ie": "nightwatch --config nightwatch.js --env ie",
     "browser-test-remote-safari": "nightwatch --config nightwatch.js --env safari",
-    "build": "mkdir -p build; browserify src/index.js -g yo-yoify -o build/app.js; babel --plugins babel-plugin-transform-es2015-template-literals,babel-plugin-transform-es2015-block-scoping build/app.js --out-file build/app.js",
+    "build": "mkdir -p build; browserify src/index.js -g yo-yoify -o build/app.js; babel --plugins babel-plugin-transform-es2015-template-literals,babel-plugin-transform-es2015-block-scoping build/app.js --out-file build/app.js; lessc --clean-css assets/less/styles.less build/app.css",
```
Comment (axic) [inline package.json]: Don't use `less --clean-css` because that compacts the file and `csslint` will not be very useful.

**Comment by axic** [comment]
I have made these changes at (just to try it myself out): https://github.com/ethereum/browser-solidity/tree/use-less

@jcbrmn if you want you can base any future work off that branch.

**Comment by jcbrmn** [comment]
Had a busy week and will take a look at these changes this weekend. Will probably just create a new PR to keep things cleaner if thats alright.

**Comment by axic** [comment]
@jcbrmn replaced this by #312. You can open pull requests aginst that branch (`use-less`). Thank you for the work so far!

**Pull request closed without merging**